projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -325,7 +324,7 @@ export const getStepContent = ( state, stepSlug ) => {
 					'With Jetpack Newsletter you can keep your audience engaged by automatically sending your content via email. <ExternalLink>Learn more</ExternalLink>',
 					'jetpack'
 				),
-				descriptionLink: 'https://jetpack.com/newsletter/',
+				descriptionLink: getRedirectUrl( 'jetpack-newsletter-landing' ),
```

Comment:
I see that you haven't changed the URL for this redirect link, but the URL `https://jetpack.com/newsletter` is a 404 for me. Do you know if that's still being worked on or if we should redirect somewhere else?

Reply:
Yes, it's being worked on; the team aims to have it ready for release day.
pbNhbs-7mF-p2